### ISSUE TITLE
Should ENV be process.env.NODE_ENV in documentation?

### ISSUE THREAD

**Issue by jkup** [reactjs/react-a11y#130]
Hey!

Under the usage documentation it says:

```
var a11y = require('react-a11y');
if (ENV === 'development') a11y(React);
```

but that global isn't defined right? Should the docs say to use process.env.NODE_ENV or something instead? I can make a PR but I wanted to make sure I'm not missing something.

**Comment by janbiasi** [comment]
I agree. I think that they actually meant the NODE_ENV which is the ENV of the React Application 😄 👍

**Comment by erin-doyle** [comment]
Yep you're right, we will fix that!
